modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/resource/v1_0/NotificationQueueEntryResourceImpl.java

```diff
@@ -73,7 +73,7 @@ public Page<NotificationQueueEntry> getNotificationQueueEntriesPage(
 				"get",
 				addAction(
 					ActionKeys.VIEW, "getNotificationQueueEntriesPage",
-					NotificationConstants.RESOURCE_NAME,
+					NotificationConstants.RESOURCE_NAME_NOTIFICATION_QUEUE,
```

Comment:
@carolmariaabb I think we can't add this key in the Map because it isn´t possible create NotificationQueueEntry in UI. If we added this key, a option in kebab menu is show. What do you think?

Reply:
@murilostodolni hmm, I thought that it wasn't related. Did you add it and the option appeared in the UI?  I thought that this other [map](https://github.com/liferay-objects/liferay-portal/blob/c51689b8dde4b910b6377ce437706d05236fac86/modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/resource/v1_0/NotificationQueueEntryResourceImpl.java#L165-L196) was related to the options kebab..

Reply:
I tested in the objectDefinition endpoint and the "get" don't return, too. See:

![screenshot-1684937959](https://github.com/liferay-objects/liferay-portal/assets/48572495/fbfbf270-3487-42ca-9c5c-6e2e14698742)

Reply:
In this case, I will add the "create" in the map. Thank you @carolmariaabb